docs/concepts/what-is-a-retry-policy.md

```diff
@@ -7,43 +7,26 @@ tags:
   - explanation
 ---
 
-A Retry Policy is collection of attributes that instructs the Temporal Server how to retry a failure of a [Workflow Execution](/docs/concepts/what-is-a-workflow-execution) or an [Activity Task Execution](/docs/concepts/what-is-an-activity-task-execution).
-
-- If a custom Retry Policy is to be used, it must be provided as an options parameter when a Workflow Execution or an Activity Execution is spawned.
-
-- The wait time before a retry is the _retry interval_.
-  A retry interval is the smaller of two values:
-  - The [Initial Interval](#initial-interval) multiplied by the [Backoff Coefficient](#backoff-coefficient) raised to the power of the number of retries.
-  - The [Maximum Interval](#maximum-interval).
+A Retry Policy is collection of attributes that instructs the Temporal Server how to retry a failure of a [Workflow Execution](/docs/concepts/what-is-a-workflow-execution) or an [Activity Task Execution](/docs/concepts/what-is-an-activity-task-execution). Retry Policies do not apply to [Workflow Task Executions](/docs/concepts/what-is-a-workflow-task-execution), which retry indefinitely.
```

Comment:
```suggestion
A Retry Policy is a collection of attributes that instructs the Temporal Server how to retry a failure of a [Workflow Execution](/docs/concepts/what-is-a-workflow-execution) or an [Activity Task Execution](/docs/concepts/what-is-an-activity-task-execution).
Retry Policies do not apply to [Workflow Task Executions](/docs/concepts/what-is-a-workflow-task-execution), which retry indefinitely.
```

Reply:
The hard line break seems so weird to me. I imagine this is a standard thing we do? Why? 😄

Reply:
We adopted this awhile ago.
Each sentence in a Markdown file is on its own line.
It reduces wrapping while editing, and enables the scope of a change to be just the sentence rather than a whole paragraph.